sync/class.jetpack-sync-functions.php

```diff
@@ -274,6 +274,53 @@ public static function get_plugins() {
 		return apply_filters( 'all_plugins', get_plugins() );
 	}
 
+	/**
+	 * Get custom action link tags that the plugin is using
+	 * Ref: https://codex.wordpress.org/Plugin_API/Filter_Reference/plugin_action_links_(plugin_file_name)
+	 * @return array of plugin action links (key: link name value: url)
+	 */
+	public static function get_plugins_action_links( $plugin_file_singular = null ) {
```

Comment:
this could be a little bit expensive to calculate, should we add a cache?

Reply:
We don't have one now. And there is already a cache no? We only check for changes every minute. How long do you think the cache should be in place there?

Reply:
maybe daily?

Reply:
Good idea I will invalidate it on every plugin update and plugin activation.